scripts/run_notebooks/runner.py

```diff
@@ -0,0 +1,50 @@
+"""Script to run all notebooks in the docs/source/notebooks directory."""
+
+import logging
+from pathlib import Path
+
+import papermill
+from joblib import Parallel, delayed
+from tqdm import tqdm
+
+KERNEL_NAME: str = "python3"
+NOTEBOOKS_PATH = Path("docs/source/notebooks")
+NOTEBOOKS_SKIP: list[str] = [
+    "mmm_budget_allocation_example.ipynb",  # This works locally but not on GitHub Actions
```

Comment:
It seems a problem when reading the model file. I want to investigate in a future PR as this way we are already cheking most of the notebooks